.travis.yml

```diff
@@ -107,7 +107,7 @@ jobs:
         - cargo doc --verbose --no-deps
         - rustup component add clippy
         - cargo clippy --version
-        - cargo clippy -- -D warnings --verbose
+        - cargo clippy -- -D warnings --verbose -A clippy::wrong-self-convention
```

Comment:
This should be added to clippy.toml instead.

Reply:
I'm not sure how to do it.

Reply:
Oh, that would be because I told you the wrong thing. Clippy.toml is the wrong place, it should actually be inline in lib.rs [like so](https://github.com/xiph/rav1e/blob/master/src/lib.rs#L39).

Reply:
Ok, addressed, thank you :)